eo-maven-plugin/src/main/java/org/eolang/maven/RegisterMojo.java

```diff
@@ -108,6 +108,7 @@ public final class RegisterMojo extends SafeMojo {
 
     @Override
     public void exec() throws IOException {
+        this.verifyPrerequisite();
```

Comment:
@volodya-lombrozo check this one: https://www.yegor256.com/2015/12/08/temporal-coupling-between-method-calls.html